easybuild/framework/easyblock.py

```diff
@@ -897,8 +895,11 @@ def obtain_file(self, filename, extension=False, urls=None, download_filename=No
                     source_urls = []
                 source_urls.extend(self.cfg['source_urls'])
 
-                # add https://sources.easybuild.io as fallback source URL
-                source_urls.append(EASYBUILD_SOURCES_URL + '/' + os.path.join(name_letter, location))
+                # Insert --extra-source-urls command line option to the
+                # urls to try to download from.
```

Comment:
As mentioned earlier this is not only a CLI option, but can be from 2 other places. I suggest something like:

```suggestion
                # Add additional URLs as configured
```